docs/docs/workflows/built-in-functions/README.md

```diff
@@ -59,4 +59,38 @@ These URLs are specific to a single execution of your workflow. While the workfl
 - Hitting the `cancel_url` will immediately cancel that execution
 - Hitting the `resume_url` will immediately resume that execution early
 
-If you use [`$.flow.delay`](/code/nodejs/delay/), you can send these URLs to your own system to handle cancellation / resumption. You can even email your customers to let them cancel / resume workflows that run on their behalf.
\ No newline at end of file
+If you use [`$.flow.delay`](/code/nodejs/delay/), you can send these URLs to your own system to handle cancellation / resumption. You can even email your customers to let them cancel / resume workflows that run on their behalf.
+
+## Filter
+
+Using filters within your workflow allows you to only proceed when certain conditions are met. You can configure specific criteria that must be met, as well as whether you'd like to stop or proceed with your workflow. Filters are a simple yet powerful tool for customizing your workflow, and can be used two few different ways:
```

Comment:
"and can be used two few different ways"

Reply:
boops